### ISSUE TITLE
Browser warning about leaving the page when uploading two videos (presentation + presenter)

### ISSUE THREAD

**Issue by abias** [Opencast-Moodle/moodle-block_opencast#181]
Steps to reproduce:
* Go to a course
* Go to /blocks/opencast/addvideo.php
* Set a video title
* Upload just a presenter video and leave the presentation filepicker empty
* Click 'Save changes'
* Verify that your browser does not complain about anything and that the video is uploaded.
* Go to /blocks/opencast/addvideo.php again
* Set a video title
* Upload just a presentation video and leave the presenter filepicker empty
* Click 'Save changes'
* Verify that your browser does not complain about anything and that the video is uploaded.
* Go to /blocks/opencast/addvideo.php again
* Set a video title
* Upload a presentation video and a presenter video to each filepicker
* Click 'Save changes'

Expected result:
* Your browser does not complain about anything and that the video is uploaded.

Actual result (at least on Firefox/Mac):
* The browser complains that you are leaving the page and that you have unsaved changes:
![grafik](https://user-images.githubusercontent.com/1092118/117433583-ec6db780-af2b-11eb-8810-ce8f89fe9214.png)

**Comment by TamaraGunkel** [comment]
Hi,
I tried to reproduce the issue with Chrome/mac and I didn't get any errors. 
I'm not sure if it's worth investigating further because doesn't seem to be a general issue and needs many steps to reproduce.
Is that something you just stumbled across or did you hear this issue from other persons, too?

**Comment by abias** [comment]
Hi @TamaraGunkel , 

thanks for having a look.

> I tried to reproduce the issue with Chrome/mac and I didn't get any errors.

When I reported the issue, I didn't test with Chrome. But I saw this error message on Firefox/Mac multiple times during several weeks when I was testing the video upload functionality.

However, I have to admit, that the problem seems to be gone today on Firefox/Mac (89.0.1). 
As we didn't change the block_opencast plugin in production for multiple months now and just did minor Moodle core updates, I assume that this might have been a Firefox browser glitch.

> Is that something you just stumbled across or did you hear this issue from other persons, too?

It is something than I stumbled across myself. However, it was quite easy to trigger this issue by just uploading two videos into one mediapackage.

> I'm not sure if it's worth investigating further because doesn't seem to be a general issue and needs many steps to reproduce.

The long list of steps to reproduce was just to clarify what worked for me and what did not work.
It could have been simplified to 'Upload to videos into one mediapackage and you will see the warning message'.

However, as the problem is gone for now, I will close this issue.

Cheers,
Alex
